### ISSUE TITLE
Daikin Residential - No devices 

### ISSUE THREAD

**Issue by ckortenhorst** [rospogrigio/daikin_residential#62]
Have installed integration but now just showing nothing but blank integration. Not seeing anything in install instructions. 

![image](https://user-images.githubusercontent.com/5760635/158083075-61128183-a702-4139-817a-2274ee6818ba.png)

**Comment by jwillemsen** [comment]
Can you download and attach the diagnostics of this integration (see https://www.home-assistant.io/integrations/diagnostics/ for some generic info)

**Comment by rospogrigio** [comment]
> Can you download and attach the diagnostics of this integration (see https://www.home-assistant.io/integrations/diagnostics/ for some generic info)

...and please check/obfuscate any personal information you will find in the diagnostic file.

**Comment by Shogun1978** [comment]
Same for me. Altherma Geo 3 with latest firmware. I attached the diagnostics file. Maybe I am doing something wrong...?

[altherma_diag.json.txt](https://github.com/rospogrigio/daikin_residential/files/8402230/altherma_diag.json.txt)
d

**Comment by jwillemsen** [comment]
For the Altherma you need to use a different integration, see https://github.com/speleolontra/daikin_residential_altherma

**Comment by Shogun1978** [comment]
Pity... I already tried this branch too and it also delivers the same result. I hoped with this version I would be luckier :-( Thanks for the fast reply!

**Comment by patagonista** [comment]
Hello, 

When adding our airco via the Daikin residential controller I see 9 entities and after restart there are 10 entities.
A few weeks ago I was able to control the airco via HA. Now it is still possible, but cannot see the temperature in my dashboard or I cannot set the temperature.  

I get a message saying: failed to call service/set_hvac_mode. 'value'

I'm running HA on a raspberry Pi3+ and also tested this on HA in docker on Synology.
I'm not sure but three weeks ago I've set a holiday period via the Daikin iOS app.


[config_entry-daikin_residential-eaa3de1565fc4b7ca187f48acfa9ff84.json.txt](https://github.com/rospogrigio/daikin_residential/files/8499751/config_entry-daikin_residential-eaa3de1565fc4b7ca187f48acfa9ff84.json.txt)
